imap_processing/tests/ultra/unit/test_ultra_l1a.py

```diff
@@ -59,15 +87,152 @@ def test_xarray_aux(decom_ultra_aux, aux_test_path):
     assert shcoarse_attr == expected_shcoarse_attr
 
 
-def test_cdf_aux(ccsds_path, xtce_path, tmp_path, decom_ultra_aux):
-    """Tests that CDF file is created and contains same attributes as xarray_aux."""
+def test_xarray_rates(decom_ultra_rates, rates_test_path):
+    """This function checks that a xarray was
+    successfully created from the decom_ultra_rates data."""
+
+    dataset = xarray(decom_ultra_rates, ULTRA_RATES.apid[0])
+
+    # Spot check metadata data and attributes
+    specific_epoch_data = dataset.sel(Epoch="2022-05-30T22:52:00.184000")["START_RF"]
+    startrf_list = specific_epoch_data.values.tolist()
+    startrf_attr = dataset.variables["START_RF"].attrs
+
+    expected_startrf_attr = dataclasses.replace(
+        ultra_cdf_attrs.ultra_metadata_attrs,
+        catdesc="START_RF",
+        fieldname="START_RF",
+        label_axis="START_RF",
+    ).output()
```

Comment:
How much should I be commenting on the internals of the CDFs in these reviews do you all think? For example, I know the SPDF would complain about these catdesc, fieldname, label_axis. But then again, our priority in the coming months should probably focus on the actual algorithms themselves?

Reply:
I think you should flag it immediately when you see it! Much better to fix these things earlier and then others on the team can keep iterating and fixing their data products as we go. I think you should be as detailed as possible in your reviews, it is super helpful!

Reply:
@bryan-harter yes please comment at will! I would rather get it right the first time rather than go back again later. There will likely be enough of that already!